lib/contentful-management.ts

```diff
@@ -61,14 +65,25 @@ function createClient(
     defaults?: DefaultParams
   }
 ): PlainClientAPI
+function createClient<
+  T extends (ReadonlyArray<string> | readonly ['workflows']) &
+    { [K in keyof T]: { [P in K]: 'workflows' } }[number]
+>(
+  params: ClientOptions,
+  opts: {
+    type: 'plain'
+    alphaFeatures: T
+    defaults?: DefaultParams
+  }
+): PlainClientAPI & AlphaWorkflowExtension
```

Comment:
Does it mean that when someone tries to use it they'd have to manually cast it to `as PlainClientAPI & AlphaWorkflowExtension`? Perhaps it would be better to define a type for it?

Reply:
![Screenshot 2021-12-14 at 15 17 00](https://user-images.githubusercontent.com/9327071/146206570-fdc7f68f-00e8-4c3a-87a1-3e9314bdbf4b.png)
![Screenshot 2021-12-14 at 15 17 13](https://user-images.githubusercontent.com/9327071/146206575-f6f1c52b-e2df-4b6b-b832-e7dd01065dab.png)
When using the alpha client, you don't need to define it manually. As you can see in the screenshot, it will infer the correct type.